db/migrate/20170626141302_drop_public_body_translated_columns.rb

```diff
@@ -0,0 +1,54 @@
+# -*- encoding : utf-8 -*-
+class DropPublicBodyTranslatedColumns < ActiveRecord::Migration
+  def up
+    remove_column :public_bodies, :name
+    remove_column :public_bodies, :short_name
+    remove_column :public_bodies, :request_email
+    remove_column :public_bodies, :url_name
+    remove_column :public_bodies, :notes
+    remove_column :public_bodies, :first_letter
+    remove_column :public_bodies, :publication_scheme
+  end
+
+  def down
```

Comment:
Looks good, although I think we can tidy the `up` translation a bit by doing something like:

```diff
-          fields = record.translated_attributes
-          fields.each do |attribute_name, attribute_type|
+          attribute_names = record.translated_attributes_names
+          attribute_names.each do |attribute_name|
```

...as the `attribute_type` doesn't appear to have an effect